packages/wrangler/src/deploy/deploy.ts

```diff
@@ -737,6 +738,9 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 					})
 				);
 
+				logger.log("Worker Startup Time: ", result.startup_time_ms, "ms");
```

Comment:
In all the tests this value is `undefined`. That doesn't seem right?
Are those tests providing a mock response for the deployment?
Does that mock need to be updated with the startup_time_ms?

Reply:
Updated.